lib/board.rb

```diff
@@ -26,4 +26,9 @@ def format_board
     formatted_board = @grid.values.transpose
     formatted_board.unshift(@grid.keys).map {|row| row.join}
   end
+
+  def update_board(player_selection, computer_selection)
+    @grid[player_selection][@grid[player_selection].rindex('.')] = 'X'
+    @grid[computer_selection][@grid[computer_selection].rindex('.')] = 'O'
+  end
```

Comment:
The .rindex method seems like a great way to approach this - so cool! I'm assuming and noting for our reference later that we aren't checking for any 'nil' returns here because we are handling checking for available column space in the get_selection methods in the Game class.

Reply:
Additionally, it looks like we could easily break this out into two methods if we decide we want to update the board with the player's selection and update the board with the computer's selection at different times.